authority-portal-frontend/src/app/pages/catalog-page/property-grid/property-grid.component.html

```diff
@@ -25,7 +26,7 @@
     <!-- Property Value (URL) -->
     <a
       *ngIf="prop.url"
-      class="link break-all"
+      class="link break-all text-gray-700"
```

Comment:
Is this how we style all links in the AP? If not, seems a little like a pointless change to me because the visual difference between the black text and gray-700 link is negligible